### PR TITLE
[4.0] language manifest

### PR DESCRIPTION
**Pull request by brianteeman**
After the language files were renamed the xml needs to be updated as well. (I still don't see why we have the files in here anyway but thats another topic)

#### Testing Instructions
code review

Components were updated in #27430

### CODE REVIEW

File: administrator/modules/mod_multilangstatus/mod_multilangstatus.xml [hunk 1]
```diff
@@ -16,7 +16,7 @@
 		<folder>tmpl</folder>
 	</files>
 
-	<languages>
+	<languages folder="admin">
 		<language tag="en-GB">language/en-GB/en-GB.mod_multilangstatus.ini</language>
```
Comment (Quy) [inline administrator/modules/mod_multilangstatus/mod_multilangstatus.xml]: Remove `en-GB.`

**Comment by infograf768** [comment]
I do not understand this PR. None of these xtensions in core contain an admin or site folder containing language files.

So, as the prefix of the language is now unused, we indeed need a change, but not by adding a folder.
instead of changing to 
```
	<languages folder="admin">
		<language tag="en-GB">language/en-GB/mod_custom.ini</language>
		<language tag="en-GB">language/en-GB/mod_custom.sys.ini</language>
	</languages>
```
Should'nt we have
 ```
	<languages>
		<language tag="en-GB">mod_custom.ini</language>
		<language tag="en-GB">mod_custom.sys.ini</language>
	</languages>
```
@Bakual ?

**Comment by Bakual** [comment]
As Brian noted, it doesn't really matter what is written there as it is not used anyway.
Since my own extensions all ship with the language files in their extension folder, I can't even say how it should be written if the languages go into the global folder. I just put a `<folder>language</folder>` tag in my XML 😄 

Since currently some XML have a path and some not, it certainly makes sense to do it all the same now. I just assume Brian had a look first which variant would be more correct.

**Comment by brianteeman** [comment]
exactly

**Comment by astridx** [comment]
@brianteeman May I ask you why you chose the longer variant. 

When I looked at this PR, I thought the same thing as @infograf768. Therefore I would be interested in your reasons.

**Comment by brianteeman** [comment]
I followed the exact same syntax as used for components

**Comment by wilsonge** [comment]
Can you merge 4.0-dev into here to get drone rerunning please

**Comment by brianteeman** [comment]
Drone is all green now

File: administrator/modules/mod_custom/mod_custom.xml [hunk 2]
```diff
@@ -16,9 +16,9 @@
 		<filename module="mod_custom">mod_custom.php</filename>
 		<folder>tmpl</folder>
 	</files>
-	<languages>
-		<language tag="en-GB">en-GB.mod_custom.ini</language>
-		<language tag="en-GB">en-GB.mod_custom.sys.ini</language>
+	<languages folder="admin">
```
Comment (wilsonge) [inline administrator/modules/mod_custom/mod_custom.xml:19]: Sorry to be the bearer - but for plugins and modules we don't need the extra folder attribute here - components only where they work across multiple clients :(

Comment (brianteeman) [reply administrator/modules/mod_custom/mod_custom.xml:19]: I "thought" the folder referred to the folder on the site

Also we have modules of the same name that exist in both site and admin but different content

Comment (zero-24) [reply administrator/modules/mod_custom/mod_custom.xml:19]: I would expect this kind of thing is handled here: https://github.com/joomla/joomla-cms/blob/staging/administrator/modules/mod_custom/mod_custom.xml#L2

When the client is site or admin.

The folder you use here is just the source folder not the destination ;-)

**Comment by brianteeman** [comment]
@richard67 so it looks like the component version of this PR was merged and accepted but this one was closed

**Comment by richard67** [comment]
> @richard67 so it looks like the component version of this PR was merged and accepted but this one was closed

@brianteeman Seems so .. but it was you who has closed it. What is right is that the "folder" attribute does not really belong to a file system folder but to the "folder" attribute for plugins like we have it in the extensions table in database. So these changes in this PR were wrong, but for sure we should fix the language file names. Do you wanna make a new PR for it?

**Comment by brianteeman** [comment]
Will do later.

Need to get my head around the folder

**Comment by richard67** [comment]
@brianteeman Look into the extensions helper or into the insert into extensions statements in base.sql and you will quickly get what the folder is.

**Pull request closed without merging**